docs/static/monitoring/monitoring-ea-intro.asciidoc

```diff
@@ -11,18 +11,21 @@ Plus you have the option to manage all of your monitoring agents from a central
 types of data to a host. Each agent has a single policy you can update to add
 integrations for new data sources, security protections, and more.
 
-You can use {agent} to collect monitoring data for:
+You can use {agent} to collect {ls} monitoring data on:
 
-* <<dashboard-monitoring-with-elastic-agent,Dashboards>>. {agent} collects
+* <<serverless-monitoring-with-elastic-agent,{serverless-full} dashboards>>. {agent} collects
```

Comment:
WDYT about putting Elastic cloud/self-managed first, at least until serverless is GA?